### ISSUE TITLE
graph -module-depth doesn't do anything

### ISSUE THREAD

**Issue by joshgubler** [hashicorp/terraform#23750]
### Terraform Version

```
Terraform v0.12.18
```

### Terraform Configuration Files

main.tf
```hcl
module "a" {
	source = "./module_a"
}
output "out" {
	value = module.a.out
}
```
module_a/main.tf
```hcl
module "b" {
	source = "../module_b"
}
output "out" {
	value = module.b.out
}
```
module_b/main.tf
```hcl
output "out" {
	value = "test"
}
```

### Expected Behavior

`terraform graph` should output a lot of lines
`terraform graph -module-depth=1` should output less lines
`terraform graph -module-depth=0` should output even less lines

### Actual Behavior

All commands output the same lines. `-module-depth` doesn't do anything.

### Steps to Reproduce

1. `terraform init`
2. `terraform graph`
3. `terraform graph -module-depth=1`
4. `terraform graph -module-depth=0`

**Comment by apparentlymart** [comment]
Thanks for reporting this, @joshgubler!

Indeed, this `-module-depth` argument is a vestige of earlier Terraform versions and was supposed to have been removed circa Terraform 0.8 or so when the graph structure was changed to no longer have an explicit awareness of individual modules. At about that time we also removed a similar option from `terraform plan`, but apparently the cleanup was not thorough enough.

For the sake of this issue the immediate problem to fix is that we need to remove the remaining vestiges of this old option to avoid confusion. A later change could potentially re-introduce something like this feature, but that would require a more extensive rework of the `terraform graph` implementation that we won't be able to undertake in the near future.
